### PR TITLE
{bio}[dummy] fastqc 0.11.2 + 0.11.3

### PR DESCRIPTION
**Pull request by boegel**

### CODE REVIEW

**Comment by boegel** [comment]
@ehiggs: please review

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 1 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/5d5d88cf32732c4d5853 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in this PR)
Linux SL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/b25c6fa7a38e41e1f7a3 for a full test report.

File: easybuild/easyconfigs/f/fastqc/fastqc-0.11.3.eb [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/fastqc/']
+sources = ['%(name)s_v%(version)s.zip']
+
+dependencies = [('Java', '1.7.0_80')]
+
+postinstallcmds = ["chmod -R a+rx %(installdir)s/fastqc %(installdir)s/run_fastqc.bat"]
```
Comment (ehiggs) [inline easybuild/easyconfigs/f/fastqc/fastqc-0.11.3.eb]: Why are we using linux commands to make windows scripts executable?

File: easybuild/easyconfigs/f/fastqc/fastqc-0.11.2.eb [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+description = """A set of tools (in Java) for working with next generation sequencing data in the BAM format."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://www.bioinformatics.babraham.ac.uk/projects/fastqc/']
+sources = ['%(name)s_v%(version)s.zip']
+
+dependencies = [('Java', '1.7.0_60')]
+
+postinstallcmds = ["chmod -R a+rx %(installdir)s/fastqc %(installdir)s/run_fastqc.bat"]
```
Comment (ehiggs) [inline easybuild/easyconfigs/f/fastqc/fastqc-0.11.2.eb]: Why are we using linux commands to make windows scripts executable?

Comment (boegel) [inline easybuild/easyconfigs/f/fastqc/fastqc-0.11.2.eb]: we're not:

```
$ cat /apps/gent/SL6/sandybridge/software/fastqc/0.11.2/run_fastqc.bat
java -Xmx250m -classpath .;./sam-1.103.jar;./jbzip2-0.9.jar uk.ac.babraham.FastQC.FastQCApplication
```

**Comment by ehiggs** [comment]
+1 merge it

**Comment by boegel** [comment]
Thanks for the review @ehiggs !

**Pull request merged**